### ISSUE TITLE
Projections that only project for positive sending neurons. 

### ISSUE THREAD

**Issue by edaggett17** [simbrain/simbrain#51]
Hello all,

Another quick user question. I am wondering if there is a way to program static synapses to only fire when a sending neuron is positive. I realize there are neurons that will facilitate this, but for the simulation I have built I need to use decay neurons only. I have been bypassing this issue by setting the activation bounds on the neurons to restrict them to positive values only, however, I need for the neurons to be capable of negative activation values as well. 

Basically, I need synapses that will multiply their strength by the activation value of their sending neuron only if that neuron's activation is positive. If the neuron's activation is less than 0, then the value for the synapse at that step should also be zero. 

Is this functionality already built into one of the update rules? I have tinkered but haven't had any luck yet. Just thought I'd ask. 

Thank you for your help!

Eben

**Comment by edaggett17** [comment]
Thank you for your reply, Dr. Yoshimi! That sounds great! I have a Java reference guide sitting on my desk collecting dust. If you can point me in the vicinity I am certain I can figure out the scripting.

**Comment by edaggett17** [comment]
Hello,
I apologize for the delay. I am having issues getting setup to run from source though this is largely due to my clumsy navigation of Ubuntu. I am very interested in being able to do some scripting, so I will let you know as soon as I’m successful in troubleshooting my issues.

**Comment by jyoshimi** [comment]
Feel free to post whatever issues you are running into.  I am working on those wiki pages today.

**Comment by jyoshimi** [comment]
I just created this wiki page.  Let me know as issues arise.  

https://github.com/simbrain/simbrain/wiki/Creating-a-Custom-Simulation
